src/python/pcmdi/__init__.py

```diff
@@ -1,4 +1,6 @@
 import io
 from mean_climate_metrics_calculations import compute_metrics
+from mean_climate_metrics_calculations_ExtInfo import compute_metrics_ExtInfo
```

Comment:
can't let that in our repo will break evry system that doesn't have your stuff this probably needs to be done in the parameter file